File: solutions/implementation/ccc15j1.java

import java.util.*;
import java.io.*;

public class J1SpecialDay {
    public static void main (String[] args) {
        Scanner in = new Scanner(System.in);
        int m = in.nextInt(), d = in.nextInt();
        if (m>2) System.out.println("After");
        else if (m<2) System.out.println("Before");
        else {
            if (d>18) System.out.println("After");
            else if (d<18) System.out.println("Before");
            else System.out.println("Special");
        }
    }
}

File: solutions/implementation/ccc15j1.cpp
#include <iostream>

using namespace std;

int main()
{
    int month, day;
    
    //take input 
    cin >> month >> day;
    
    if(month == 1) // if january
    {
        cout << "Before" << endl;
    }
    else if(month == 2) // if feb
    {
        //need to check days 
        if(day < 18)
        {
            cout << "Before" << endl;
        }
        else if(day == 18)
        {
            cout << "Special" << endl;
        }
        else 
        {
            cout << "After" << endl;
        }
    }
    else 
    {
        cout << "After" << endl;
    }

    return 0;
}
